docker/hooks/push

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# push individual tags
+docker push $(echo $IMAGE_NAME | sed "s/:latest/:i386-latest/")
```

Comment:
As you're using bash all of these `echo|sed` tricks can be replaced by `${IMAGE_NAME/:latest/:i386-latest}` variable expansions

Reply:
New, to me. Thanks for the hint, will try!